### ISSUE TITLE
Fund the DAO

### ISSUE THREAD

**Issue by falkonprods** [syvita/daoos#45]
As a DAO member, I want to be able to fund the  DAO directly so that the  DAO  can have funds to perform work.

**Comment by cryptopanter** [comment]
First of all, all funds will go directly to treasury trust (we should deal with Treasury account in different issue).. But before the given dund goes go Treasury, it needs to het allocated some labels:
Member_fund(1)= [amount, which_person, to which proposal, to which mileston]

**Comment by asteriabtc** [comment]
this should be pretty easy. just a function to effectively donate to the dao

**Comment by 314159265359879** [comment]
Donating to the DAO is a wrapping function, right? The DAO gets STX and you get the DAO sip 010 coin?
And this is a one way street, you can get DAO coin for STX but you can not get STX for the DAO coin, right?

Members proposals and their votes decide where the STX go towards, which projects are chosen to finance with that treasury to achieve a certain goal/revenue.

**Comment by cryptopanter** [comment]
@314159265359879 I don't think we have wrapped our mind around issuing a native token for DAO contract... according to last convo, @falkonprods opinion was to seperately develop DAOos without any token for the first MVP, am I right?...otherwise we should plan for token economy.

**Comment by 314159265359879** [comment]
> 
> 
> @314159265359879 I don't think we have wrapped our mind around issuing a native token for DAO contract... according to last convo, @falkonprods opinion was to seperately develop DAOos without any token for the first MVP, am I right?...otherwise we should plan for token economy.

Ah rereading the last couple of post in https://github.com/syvita/daoos/discussions/30 I see we did conclude not needing a SIP-010 token. In scenario 1 (Freehold scenario) we do not need any token because there is a "account holder" who controls the funds on behalf of voting members. 

In scenario 2 (DAO for Open Source app creators) you want members to have voting power equal to what they brought to the table funding the DAO, right? If you wrap the asset (STX) and return an equivalent amount of an FT, that FT could help with (A) splitting up votes between members and proposals without exceeding the available funds, (B) keep an historic record (if FT's are not burned) and (C) relate _FT proportion of total to equivalent fractional treasury value_. As the treasury value decreases with funds going to projects and increases with profits from those projects: this could prove useful to tell the stx/dollar value of a vote/stake. 
Or maybe voting is the action of funding with stx, then no FT is needed? In that case "Members do not fund the DAO but they fund available projects in the DAO"?

In scenario 3 (One project/milestone and full liquidation) in this case again members could vote with STX directly.  Any member who funds with STX is a voting member by default and their stake is proportionate to the stx they staked. Full liquidation does not mean dissolving the DAO. I think this situation fits the application of DAOos I proposed here (community namespace launch) https://github.com/syvita/daoos/discussions/48.
